### DISCUSSION TITLE
Cannot add seat and update payment  method

### DISCUSSION THREAD

**Discussion by chamara8901** [community/community#39325] [Other Feature Feedback, Questions, & Ideas]
Cannot add seat and update payment  method
Question
Cannot add seat and update payment  method

          I am chamara from japan. I cannot add seat in github organization and cannot update my payment method. when i try to upade payment method there have a error like 500. How can i resolve this problem?Please help me.

**Comment by queenofcorgis** [comment]
Hi @chamara8901! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by chamara8901** [comment]
Dear Sir /Madam

  I appreciate your quick response. We are IT company in japan. Please help
me to resolve this issue emergency because we started new project. We are
project management team. I want add seat many times. They are wating to add
github organization. I cannot change my payment method and anything in
github its have error like a 500. Please help emergency.
We are wating your response.

Thank you very much.
